dist/index.html

```diff
@@ -1,12 +1 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  <div class="content"></div>
-  <script src="main.js"></script>
-</body>
-</html>
\ No newline at end of file
+<!doctype html><html><head><meta charset="utf-8"><title>Burger Shack</title><meta name="viewport" content="width=device-width,initial-scale=1"></head><body><script src="main.js"></script></body></html>
```

Comment:
You are missing `<div id="content"></div>`
![image](https://user-images.githubusercontent.com/31897434/77474856-1d6a5600-6e18-11ea-81b6-98f89d3dd4d5.png)
Kindly update the code

Reply:
- I've added the missing div tag with the 'content' id.
- And, I've turned on stickler for this repo